lib/manager/poetry/extract.ts

```diff
@@ -103,10 +114,29 @@ export function extractPackageFile(
     logger.debug(`${fileName} contains no poetry section`);
     return null;
   }
+
+  // handle the lockfile
+  const lockContents = await readLocalFile('poetry.lock', 'utf8');
```

Comment:
This assumes the lock file is always in the root of the repo. Need to get the sibling file of the package file instead

Reply:
Done!